drivers/SmartThings/zwave-thermostat/src/init.lua

```diff
@@ -87,6 +102,31 @@ local function set_setpoint_factory(setpoint_type)
   end
 end
 
+local function setpoint_capabilites_report(driver, device, cmd)
+  local args = cmd.args
+  local min_temp_c = args.min_value
+  local max_temp_c = args.max_value
+
+  local scale = 'C'
+  if args.scale1 == ThermostatSetpoint.scale.FAHRENHEIT then
+    scale = 'F'
+  end
+
+  local capability_constructor = nil
+  if args.setpoint_type == ThermostatSetpoint.setpoint_type.HEATING_1 then
+    capability_constructor = capabilities.thermostatHeatingSetpoint.heatingSetpointRange
+  else
```

Comment:
you should specifically check for `COOLING_1` since it is possible (if unlikely) that some other controller on the network inquires about the limits for one of the other setpoint types we don't support.